src/libaktualizr/telemetry/telemetryconfig.h

```diff
@@ -8,9 +8,10 @@ struct TelemetryConfig {
    * Report device network information: IP address, hostname, MAC address
    */
   bool report_network{true};
+  bool report_config{true};
 
   void updateFromPropertyTree(const boost::property_tree::ptree& pt);
   void writeToStream(std::ostream& out_stream) const;
 };
 
-#endif  // TELEMETRY_TELEMETRY_CONFIG_H_
\ No newline at end of file
+#endif  // TELEMETRY_TELEMETRY_CONFIG_H_
```

Comment:
Not specific to this particular PR, just reminded about thing I wanted to ask: is there reason why we don't use "#pragma once"?

Reply:
It's not in the standard, so why should we use it?

Reply:
It's so widely supported I though it's part of the standard already. Good to know.

Reply:
Yes, according to wikipedia it's basically supported everywhere.

But I suppose the main reason we don't use it, is just plain inertia: the benefits might or might not be offseted by the amount of work it takes to convert existing files and check that it works well for all our use cases.